src/main/java/org/broadinstitute/hellbender/utils/read/Fragment.java

```diff
@@ -11,6 +11,7 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.Optional;
```

Comment:
Is this new import used?

Reply:
Oops, I forgot to remove that before committing.  I'll take that out